code/controllers/subsystem/processing/instruments.dm

```diff
@@ -0,0 +1,56 @@
+PROCESSING_SUBSYSTEM_DEF(instruments)
+	name = "Instruments"
+	wait = 0.5
+	init_order = INIT_ORDER_INSTRUMENTS
+	flags = SS_KEEP_TIMING
+	priority = FIRE_PRIORITY_INSTRUMENTS
+	/// List of all instrument data, associative id = datum
+	var/static/list/datum/instrument/instrument_data = list()
```

Comment:
Why are all these variables static, there shouldn't ever be more than one subsystem at once and master controller reboot shouldn't keep these going implicitly.

Reply:
i don't need to define Recover() if it never needs to well, recover this. is this bad practice? was just trying to save from maintenance overhead so to speak from having to add variables to Recver()